src/calculator.js

```diff
@@ -0,0 +1,14 @@
+const calculator = (() => {
+  const add = (a, b) => a + b;
+  const subtract = (a, b) => a - b;
+  const divide = (a, b) => a / b;
+  const multiply = (a, b) => a * b;
+
+  return {
+    add,
+    subtract,
+    divide,
+    multiply,
+  };
+})();
```

Comment:
- [ ] Make it an Object using arrow function as you did with all the other ones:
```javascript
// Instead of your current function
// Lest make it an Object
const Calculator = () => {
  const add = (a, b) => a + b;
  const subtract = (a, b) => a - b;
  const divide = (a, b) => a / b;
  const multiply = (a, b) => a * b;

  return {
    add,
    subtract,
    divide,
    multiply,
  };
};
// If you have any question regarding this conversion 
// (which you did on your previews objects, let me know through slack)
```